sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -3021,4 +3022,10 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
       JdbcDialects.registerDialect(H2Dialect)
     }
   }
+
+  test("Explain shows executed SQL query") {
+    val df = sql("SELECT max(id) FROM h2.test.people WHERE id > 1")
+    val explained = getNormalizedExplain(df, FormattedMode)
+    assert(explained.contains("External engine query:"))
```

Comment:
We won't display `WHERE id > 1` because of this change? https://github.com/apache/spark/pull/45102/files#diff-ecf5b374060c1222d3a0a1295b4ec2cb5d07603db460273484b1753e1cab9f90R199

Reply:
Fixed, thank you